local_server/local_tts_server/start_kokoro_server.ps1

```
@@ -0,0 +1,571 @@
+            "misaki[zh]>=0.8.2",
+            "soundfile",
+            "numpy",
+            "https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.8.0/en_core_web_sm-3.8.0-py3-none-any.whl"
+        )
+        if ($installed.ExitCode -ne 0) {
+            throw "Failed to install Kokoro server dependencies: $($installed.Text.Trim())"
+        }
+    }
+}
+
+function Ensure-CudaTorchIfNeeded {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Ensure-CudaTorchIfNeeded' uses an unapproved verb.

(PSUseApprovedVerbs)